composer.json

```diff
@@ -17,10 +17,10 @@
             "type": "package",
             "package": {
                 "name": "danskernesdigitalebibliotek/dpl-react",
-                "version": "0.3.0-rc10",
+                "version": "0.3.0-rc11",
                 "type": "drupal-library",
                 "dist": {
-                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-react/releases/download/release-develop/dist.zip",
+                    "url": "https://github.com/reload/dpl-react/releases/download/release-fix%2Ffbi-update-take-2/dist.zip",
```

Comment:
This change should be removed from the PR. The client will be automatically updated when danskernesdigitalebibliotek/dpl-react#302 lands.